### ISSUE TITLE
microk8s enable kubeflow --bundle lite --ignore-min-mem seems to work but gives error in output

### ISSUE THREAD

**Issue by davecore82** [canonical/microk8s#2094]
Enabling kubeflow with the following options taken from the documentation (https://microk8s.io/docs/addon-kubeflow) seems to work but there is an error in the output:

```
sudo snap install microk8s --classic --channel=1.19
sudo usermod -a -G microk8s $USER
sudo chown -f -R $USER ~/.kube
# logout/log back in
microk8s status --wait-ready
microk8s enable kubeflow --bundle lite --ignore-min-mem

[...]
Waited 540s for operator pods to come up, 1 remaining.
Operator pods ready.
Congratulations, Kubeflow is now available.

The dashboard is available at http://10.64.140.43.xip.io

    Username: admin
    Password: <password>

To see these values again, run:

    microk8s juju config dex-auth static-username
    microk8s juju config dex-auth static-password



To tear down Kubeflow and associated infrastructure, run:

    microk8s disable kubeflow

usage: microk8s status [-h] [--format {yaml,pretty,short}] [-w] [-t TIMEOUT]
                       [-a ADDON] [--yaml]
microk8s status: error: argument -a/--addon: expected one argument
Nothing to do for --bundle
```

**Comment by davecore82** [comment]
This is what seems to have been deployed:

```
ubuntu@node05ob32:~$ sudo microk8s juju status
Model     Controller  Cloud/Region        Version  SLA          Timestamp
kubeflow  uk8s        microk8s/localhost  2.8.6    unsupported  14:54:04Z

App                          Version                         Status  Scale  Charm                        Store       Rev  OS          Address         Notes
admission-webhook            registry.jujucharms.com/kub...  active      1  admission-webhook            jujucharms    7  kubernetes  10.152.183.84   
argo-controller              registry.jujucharms.com/kub...  active      1  argo-controller              jujucharms  241  kubernetes                  
argo-ui                      registry.jujucharms.com/kub...  active      1  argo-ui                      jujucharms  155  kubernetes  10.152.183.138  
dex-auth                     registry.jujucharms.com/kub...  active      1  dex-auth                     jujucharms   99  kubernetes  10.152.183.203  
istio-ingressgateway         registry.jujucharms.com/ist...  active      1  istio-ingressgateway         jujucharms    8  kubernetes  10.64.140.43    
istio-pilot                  registry.jujucharms.com/ist...  active      1  istio-pilot                  jujucharms    8  kubernetes  10.152.183.105  
jupyter-controller           registry.jujucharms.com/kub...  active      1  jupyter-controller           jujucharms  253  kubernetes                  
jupyter-web                  registry.jujucharms.com/kub...  active      1  jupyter-web                  jujucharms  160  kubernetes  10.152.183.183  
katib-controller             registry.jujucharms.com/kub...  active      1  katib-controller             jujucharms  153  kubernetes  10.152.183.170  
katib-db                     rocks.canonical.com:443/mar...  active      1  mariadb-k8s                  jujucharms   34  kubernetes  10.152.183.244  
katib-manager                registry.jujucharms.com/kub...  active      1  katib-manager                jujucharms  140  kubernetes  10.152.183.80   
katib-ui                     registry.jujucharms.com/kub...  active      1  katib-ui                     jujucharms  143  kubernetes  10.152.183.216  
kubeflow-dashboard           registry.jujucharms.com/kub...  active      1  kubeflow-dashboard           jujucharms  113  kubernetes  10.152.183.172  
kubeflow-profiles            registry.jujucharms.com/kub...  active      1  kubeflow-profiles            jujucharms  116  kubernetes  10.152.183.129  
metacontroller               registry.jujucharms.com/kub...  active      1  metacontroller               jujucharms  142  kubernetes  10.152.183.44   
metadata-api                 registry.jujucharms.com/kub...  active      1  metadata-api                 jujucharms  109  kubernetes  10.152.183.71   
metadata-db                  rocks.canonical.com:443/mar...  active      1  mariadb-k8s                  jujucharms   34  kubernetes  10.152.183.16   
metadata-envoy               registry.jujucharms.com/kub...  active      1  metadata-envoy               jujucharms   94  kubernetes  10.152.183.241  
metadata-grpc                registry.jujucharms.com/kub...  active      1  metadata-grpc                jujucharms   90  kubernetes  10.152.183.109  
metadata-ui                  registry.jujucharms.com/kub...  active      1  metadata-ui                  jujucharms  106  kubernetes  10.152.183.218  
minio                        registry.jujucharms.com/kub...  active      1  minio                        jujucharms  160  kubernetes  10.152.183.207  
oidc-gatekeeper              registry.jujucharms.com/kub...  active      1  oidc-gatekeeper              jujucharms   99  kubernetes  10.152.183.111  
pipelines-api                registry.jujucharms.com/kub...  active      1  pipelines-api                jujucharms  164  kubernetes  10.152.183.69   
pipelines-db                 rocks.canonical.com:443/mar...  active      1  mariadb-k8s                  jujucharms   34  kubernetes  10.152.183.19   
pipelines-persistence        registry.jujucharms.com/kub...  active      1  pipelines-persistence        jujucharms  247  kubernetes                  
pipelines-scheduledworkflow  registry.jujucharms.com/kub...  active      1  pipelines-scheduledworkflow  jujucharms  244  kubernetes                  
pipelines-ui                 registry.jujucharms.com/kub...  active      1  pipelines-ui                 jujucharms  159  kubernetes  10.152.183.235  
pipelines-viewer             registry.jujucharms.com/kub...  active      1  pipelines-viewer             jujucharms  178  kubernetes                  
pipelines-visualization      registry.jujucharms.com/kub...  active      1  pipelines-visualization      jujucharms   93  kubernetes  10.152.183.239  
pytorch-operator             registry.jujucharms.com/kub...  active      1  pytorch-operator             jujucharms  248  kubernetes                  
seldon-core                  registry.jujucharms.com/kub...  active      1  seldon-core                  jujucharms   98  kubernetes  10.152.183.33   
tf-job-operator              registry.jujucharms.com/kub...  active      1  tf-job-operator              jujucharms  239  kubernetes                  

Unit                            Workload  Agent  Address       Ports                                                                                                  Message
admission-webhook/0*            active    idle   10.1.227.202  443/TCP                                                                                                
argo-controller/0*              active    idle   10.1.227.236                                                                                                         
argo-ui/0*                      active    idle   10.1.227.206  8001/TCP                                                                                               
dex-auth/2*                     active    idle   10.1.228.10   5556/TCP,15090/TCP                                                                                     
istio-ingressgateway/0*         active    idle   10.1.227.248  15020/TCP,80/TCP,443/TCP,15029/TCP,15030/TCP,15031/TCP,15032/TCP,15443/TCP,15011/TCP,8060/TCP,853/TCP  
istio-pilot/0*                  active    idle   10.1.227.216  8080/TCP,15010/TCP,15012/TCP,15017/TCP                                                                 
jupyter-controller/1*           active    idle   10.1.227.217                                                                                                         
jupyter-web/0*                  active    idle   10.1.227.213  5000/TCP                                                                                               
katib-controller/0*             active    idle   10.1.227.214  443/TCP,8080/TCP                                                                                       
katib-db/0*                     active    idle   10.1.227.231  3306/TCP                                                                                               ready
katib-manager/0*                active    idle   10.1.228.4    6789/TCP                                                                                               
katib-ui/0*                     active    idle   10.1.227.233  8000/TCP                                                                                               
kubeflow-dashboard/0*           active    idle   10.1.228.3    8082/TCP                                                                                               
kubeflow-profiles/0*            active    idle   10.1.227.246  8080/TCP,8081/TCP                                                                                      
metacontroller/0*               active    idle   10.1.227.238  9999/TCP                                                                                               
metadata-api/0*                 active    idle   10.1.227.215  8080/TCP                                                                                               
metadata-db/0*                  active    idle   10.1.227.251  3306/TCP                                                                                               ready
metadata-envoy/0*               active    idle   10.1.228.1    9090/TCP,9091/TCP                                                                                      
metadata-grpc/0*                active    idle   10.1.228.0    8080/TCP                                                                                               
metadata-ui/0*                  active    idle   10.1.228.5    3000/TCP                                                                                               
minio/0*                        active    idle   10.1.227.232  9000/TCP                                                                                               
oidc-gatekeeper/1*              active    idle   10.1.228.8    8080/TCP,15090/TCP                                                                                     
pipelines-api/0*                active    idle   10.1.227.240  8887/TCP,8888/TCP                                                                                      
pipelines-db/0*                 active    idle   10.1.227.227  3306/TCP                                                                                               ready
pipelines-persistence/0*        active    idle   10.1.227.237                                                                                                         
pipelines-scheduledworkflow/0*  active    idle   10.1.227.230                                                                                                         
pipelines-ui/0*                 active    idle   10.1.227.245  3000/TCP                                                                                               
pipelines-viewer/0*             active    idle   10.1.227.229                                                                                                         
pipelines-visualization/0*      active    idle   10.1.227.235  8888/TCP                                                                                               
pytorch-operator/0*             active    idle   10.1.227.197                                                                                                         
seldon-core/0*                  active    idle   10.1.228.7    8080/TCP,9876/TCP                                                                                      
tf-job-operator/0*              active    idle   10.1.228.6   
```                                                                                                        

Attaching microk8s inspect tarball as well
[inspection-report-20210315_145123.tar.gz](https://github.com/ubuntu/microk8s/files/6142255/inspection-report-20210315_145123.tar.gz)

**Comment by davecore82** [comment]
I also tried with an extra `--` but I still get an error:

```
microk8s enable kubeflow -- --bundle lite --ignore-min-mem
[...]
Congratulations, Kubeflow is now available.

The dashboard is available at http://10.64.140.43.xip.io

    Username: admin
    Password: <password>

To see these values again, run:

    microk8s juju config dex-auth static-username
    microk8s juju config dex-auth static-password

To tear down Kubeflow and associated infrastructure, run:

    microk8s disable kubeflow

usage: microk8s status [-h] [--format {short,pretty,yaml}] [-w] [-t TIMEOUT]
                       [-a ADDON] [--yaml]
microk8s status: error: argument -a/--addon: expected one argument
Nothing to do for --
```

**Comment by aicanhelp** [comment]
It seems there is a bug. When I input the command 'microk8s enable kubeflow --bundle lite', nothing to do for '--bundle lite'. But when I input the command 'microk8s enable kubeflow:'--bundle lite'', it works.
